pandas/core/groupby/ops.py

```diff
@@ -531,6 +531,16 @@ def _call_cython_op(
                     result_mask=result_mask,
                     is_datetimelike=is_datetimelike,
                 )
+            elif self.how in ["add"]:
```

Comment:
can we share code with L523 (ok if its a followon)

Reply:
not ATM, L523 funcs have mask args that this doesnt (for now at least)

Reply:
in #43467, `mask` and `result_mask` parameters were added to `_libs.groupby.group_mean` so that the same call could be made to `mean` as ` min` and `max`. For consistency, should either remove those unused parameters from `_libs.groupby.group_mean` or add the unused parameters to `_libs.groupby.group_add` also.